### DISCUSSION TITLE
Device Polling from Ring Acct - passing into Home Assistant

### DISCUSSION THREAD

**Discussion by crl0wryjr** [tsightler/ring-mqtt#786] [Support Q&A] [answered]
Today I added a new ZWAVE sensor to my Ring account, and it did not automatically appear in Home Assistant. Looking at the logs on the ring-zwave docker container, I didn't see the device there either. I restarted the Docker container for ring-mqtt and boom --- I now show 17 devices.   

Do I need to need to restart the ring-mqtt container every time I add a new device to my ring account?
Is this related to the device being a 3rd part ZWAVE device?

Thanks
Rob

**Comment by tsightler** [comment, marked as answer]
> Do I need to need to restart the ring-mqtt container every time I add a new device to my ring account?

Yes, ring-mqtt only discovers devices during startup so restart is required for adding/removing devices.

**Reply by crl0wryjr** [reply]
Thanks for the quick reply, and for the awesome integration.
